### PR TITLE
Impulse 2024 page

### PR DESCRIPTION
**Pull request by Vignaraj-pai**
# Description

Impulse 2024 Page created

Fixes # (issue)

## Dependencies

None

## Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- [ ] 

---

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: corpus/templates/pages/impulse.html [hunk 1]
```diff
@@ -0,0 +1,623 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block title %} Impulse {% endblock %}
+
+{% block style %}
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.4/tiny-slider.css">
+
+{% comment %} <link rel="stylesheet" href="{% static 'css/impulse.css' %}"> {% endcomment %}
```
Comment (anirudhprabhakaran3) [inline corpus/templates/pages/impulse.html:10]: Why this comment?

File: corpus/templates/pages/impulse.html [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+                <center>
+                    <!-- <p style="font-size: 1.5em; ">Prizes and goodies worth 10k!</p> -->
+                    <p style="font-size: 1.5em; ">Prizes and goodies to be announced soon !</p>
+                </center>
+            </div>
+            <!-- -------------------------------- -->
+        </div>
+    </section>
+
+    {% comment %} Schedule Section {% endcomment %}
```
Comment (anirudhprabhakaran3) [inline corpus/templates/pages/impulse.html:170]: Use timeline for schedule: https://daisyui.com/components/timeline/

File: corpus/templates/pages/impulse.html [hunk 3]
```diff
@@ -0,0 +1,623 @@
...
+                            height="22"
+                            viewBox="0 0 22 22"
+                            fill="none"
+                            xmlns="http://www.w3.org/2000/svg"
+                            >
+                            <path
+                                d="M11 15.675C10.7937 15.675 10.6219 15.6062 10.45 15.4687L2.54374 7.69998C2.23436 7.3906 2.23436 6.90935 2.54374 6.59998C2.85311 6.2906 3.33436 6.2906 3.64374 6.59998L11 13.7844L18.3562 6.53123C18.6656 6.22185 19.1469 6.22185 19.4562 6.53123C19.7656 6.8406 19.7656 7.32185 19.4562 7.63123L11.55 15.4C11.3781 15.5719 11.2062 15.675 11 15.675Z"
+                                fill="currentColor"
+                                />
+                        </svg>
```
Comment (anirudhprabhakaran3) [inline corpus/templates/pages/impulse.html:348]: If the same SVG is used everywhere, put it in a file in `static` and include it here.

**Review by anirudhprabhakaran3 — CHANGES_REQUESTED** [review]

File: corpus/templates/pages/impulse.html (hunk 1 above)
Comment (Vignaraj-pai) [reply corpus/templates/pages/impulse.html]: Removed it.

File: corpus/templates/pages/impulse.html (hunk 2 above)
Comment (Vignaraj-pai) [reply corpus/templates/pages/impulse.html:171]: Done.

File: corpus/templates/pages/impulse.html (hunk 3 above)
Comment (Vignaraj-pai) [reply corpus/templates/pages/impulse.html]: Extracted all repeated SVGs into `static` "img/symbols"

**Review by anirudhprabhakaran3 — APPROVED** [review]

**Pull request merged**